src/execution_tree/primitives/console_output.cpp

```diff
@@ -43,7 +43,7 @@ namespace phylanx { namespace execution_tree { namespace primitives
     console_output::console_output(
             std::vector<primitive_argument_type>&& operands)
       : base_primitive(std::move(operands))
-    {}
+    {} 
```

Comment:
We will add the inspect checker to this repository soon. One of the things it will flag as an error is end-of-line-whitespace. Would you mind configuring your editor such that it removes that automatically?

Reply:
No problem.